site/content/en/docs/tasks/troubleshooting/troubleshooting_jobs.md

```diff
@@ -60,6 +60,45 @@ Job is called `my-job` in the `my-namespace` namespace.
    job-my-job-19797   user-queue   cluster-queue   9m45s
    ```
 
+## Does my cluster-queue match the job resource requests?
+
+When you submit a job that has a resource request, for example:
+
+```bash
+$ kubectl get jobs job-0-9-size-6 -o json | jq -r .spec.template.spec.containers[0].resources
```

Comment:
```suggestion
$ kubectl get jobs job-0-9-size-6 -o jsonpath='.spec.template.spec.containers[0].resources'
```

Although this wouldn't pretty print

Reply:
I do like the pretty printing <3 But happy to change it if the docs already have a convention.